codebuild/bin/grep_simple_mistakes.sh

```diff
@@ -18,7 +18,7 @@ FAILED=0
 # Grep for any instances of raw memcpy() function. s2n code should instead be
 # using one of the *_ENSURE_MEMCPY macros.
 #############################################
-S2N_FILES_ASSERT_NOT_USING_MEMCPY=$(find "$PWD" -type f -name "s2n*.[ch]" -not -path "*/tests/*"  -not -path "*/pq-crypto/*")
+S2N_FILES_ASSERT_NOT_USING_MEMCPY=$(find "$PWD" -type f -name "s2n*.[ch]" -not -path "*/tests/*"  -not -path "*/pq-crypto/*" -not -path "*/s2n_ensure.[ch]")
```

Comment:
Just wondering why this is needed? It doesn't seem like any memcpys were added to s2n_ensure?

Reply:
Initially I had refactored the memcpy out of the c file to have a better chance of inlining without LTO. But that broke a bunch of cbmc and sidetrail tests and I wasn't in the mood to fix them. But I ended up keeping this in, since if there is an allowed memcpy, it would be in that file. However currently we are using a memmove (which should also be scanned, TBH)